software/wmla120.py

```diff
@@ -1671,7 +1674,7 @@ def init_clients(self):
         elif self.sw_vars['ansible_become_pass'] is None:
             cmd += '--ask-become-pass '
             prompt_msg = "\nClient password required for privilege escalation"
-        # Verification Loop
+        # Verfication Loop
```

Comment:
Typo fixed in PR #517 re-introduced here.

Reply:
fixed